preprocessing/instruments.py

```diff
@@ -7,13 +7,37 @@ def is_drums(track):
 def is_bass(track):
     if any(track.sum(axis=1) > 1):
         return False
+    elif track.sum()/len(np.nonzero(track)[0]) > 47:
```

Comment:
`track.sum()` will just give you the total number of activations, not related to the pitch. So the left hand side of this statement would grow with song length, which isn't right.